homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -249,6 +252,18 @@
     ATTR_FAN_LEVEL: "fan_level",
 }
 
+AVAILABLE_ATTRIBUTES_AIRPURIFIER_3C = {
+    ATTR_AIR_QUALITY_INDEX: "aqi",
```

Comment:
Sensor measurements should not be included as state attributes for the fan entity. They should be separate sensor entities.

Reply:
Done! I've added sensor entities for air quality and filter status